update multi-tenancy guide. Add infromation about Enterprise and Rete… (#3285)
VictoriaMetrics/VictoriaMetrics: docs/guides/guide-vmcluster-multiple-retention-setup.md

Community version of VictoriaMetrics supports only one retention period via [-retentionPeriod](https://docs.victoriametrics.com/#retention) command-line flag.
The following solution allows supporting multiple retentions for community version of VictoriaMetrics.

**Enterprise Solution**

> Enterprise version of VictoriaMetrics Single and Cluster support multi-retention from version [1.83.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.83.0).
> You need to specify `-retentionFilters` for vmstorage or single instances. You can set more than one filter 
> 
> Example:

Reviewer: ```suggestion
Enterprise version of VictoriaMetrics Single and Cluster support multi-retention from version [1.83.0](https://github.com/VictoriaMetrics/VictoriaMetrics/releases/tag/v1.83.0) via `-retentionFilters` command-line flag. For example:
```

* VictoriaMetrics Single - `-retentionFilter='{env=~"dev|staging"}:30d' -retentionPeriod=1y`

> * VictoriaMetrics Cluster - `-retentionFilter='{vm_account_id=~"42.*"}:1d' -retentionFilter='{env=~"dev|staging"}:3d' -retentionPeriod=4w`. It supports retention per tenant

Reviewer: ```suggestion
* VictoriaMetrics Cluster - `-retentionFilter='{vm_account_id=~"42.*"}:1d' -retentionFilter='{env=~"dev|staging"}:3d' -retentionPeriod=4w`. Labels `vm_account_id` and `vm_project_id` can be used to specify tenant and project respectively.
```

> Documentation for the single version is [here](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#multiple-retentions), and for the cluster - [here](https://docs.victoriametrics.com/Cluster-VictoriaMetrics.html#retention-filters)

Reviewer: ```suggestion
Multi-retention is explained in more detail in corresponding docs for [single-node](https://docs.victoriametrics.com/Single-server-VictoriaMetrics.html#multiple-retentions) and [cluster](https://docs.victoriametrics.com/Cluster-VictoriaMetrics.html#retention-filters) versions.
```
